vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporter.java

```diff
@@ -220,8 +234,13 @@ private VCFHeader getVcfHeaderFilteringInfoLines(String headerObject) throws IOE
         ByteArrayInputStream bufferedInputStream = new ByteArrayInputStream(headerObject.getBytes());
         LineIterator filteringLineIterator = new VcfHeaderFilteringLineIterator(bufferedInputStream, "FILTER", "FORMAT",
                                                                                 "INFO");
-        FeatureCodecHeader featureCodecHeader = vcfCodec.readHeader(filteringLineIterator);
-        return (VCFHeader) featureCodecHeader.getHeaderValue();
+        try {
+            FeatureCodecHeader featureCodecHeader = vcfCodec.readHeader(filteringLineIterator);
+            return (VCFHeader) featureCodecHeader.getHeaderValue();
+        } catch (TribbleException e) {
```

Comment:
Could we document when is this exception thrown?

Reply:
I added an extra message in the logging explaining that tribble exception is thrown when the header can't be parsed.